client/login/wp-login/index.jsx

```diff
@@ -70,14 +80,19 @@ export class Login extends React.Component {
 	}
 
 	render() {
-		const { locale, translate, twoFactorAuthType } = this.props;
-
+		const {
+			locale,
+			socialConnect,
+			translate,
+			twoFactorAuthType,
+		} = this.props;
 		const canonicalUrl = `https://${ locale !== 'en' ? locale + '.' : '' }wordpress.com/login`;
 
 		return (
 			<div>
 				<Main className="wp-login__main">
-					{ this.renderLocaleSuggestions() }
+					{ ! socialConnect &&
```

Comment:
This check is redundant, it's handled in `renderLocaleSuggestions()` (along with 2FA pages).